app/controllers/collections_controller.rb

```diff
@@ -202,7 +231,7 @@ def delete_document_from_item
 
   def edit_collection
     begin
-      collection = validate_collection(params[:id], params[:api_key])
+      collection = validate_collection(sanitize_collection_name(params[:id]), params[:api_key])
```

Comment:
`validate_collection` is not the only place that `sanitize_collection_name` is used, so it can't really be removed. However would it make more sense to make it a static method of the collection model, so that something akin to `name = Collection.sanitise_name('Some bad name')` could be used instead of having it in the collection controller as a private method?